app/javascript/components/mockBinderSleeves.js

```diff
@@ -0,0 +1,44 @@
+let mockBinder = [
+   {
+    id:1,
+    user_id:1,
+    card_id:1,
+    quantity:1,
+    favorites:true,
+
+   }, 
+   {
+    id:2,
+    user_id:1,
+    card_id:2,
+    quantity:1,
+    favorites:true,
+
+   }, 
+   {
+    id:3,
+    user_id:1,
+    card_id:3,
+    quantity:2,
+    favorites:true,
+
+   }, 
+   {
+    id:4,
+    user_id:2,
+    card_id:1,
+    quantity:1,
+    favorites:true,
+
+   }, 
+   {
+    id:5,
+    user_id:2,
+    card_id:2,
+    quantity:3,
+    favorites:true,
```

Comment:
trailing commas are fine in JS. Just be careful about when you migrate this to seeds, Ruby is not a fan of trailing commas